testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -331,7 +331,7 @@ pub fn wait_for_runloop(blocks_processed: &Arc<AtomicU64>) {
     }
 }
 
-fn wait_for_microblocks(microblocks_processed: &Arc<AtomicU64>, timeout: u64) -> bool {
+pub fn wait_for_microblocks(microblocks_processed: &Arc<AtomicU64>, timeout: u64) -> bool {
```

Comment:
while we are here, can we add a concise but precise contract for this method?

It's not clear from the signature what is going to happen.

Reply:
Added